MON-2693: Scrape profiles (#1785)
openshift/cluster-monitoring-operator: Documentation/api.md

| retentionSize | string | Defines the maximum amount of disk space used by data blocks plus the write-ahead log (WAL). Supported values are `B`, `KB`, `KiB`, `MB`, `MiB`, `GB`, `GiB`, `TB`, `TiB`, `PB`, `PiB`, `EB`, and `EiB`. By default, no limit is defined. |
| tolerations | [][v1.Toleration](https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.26/#toleration-v1-core) | Defines tolerations for the pods. |
| topologySpreadConstraints | []v1.TopologySpreadConstraint | Defines the pod's topology spread constraints. |

> | collectionProfile | CollectionProfile | Defines the collection profile which Prometheus will use to collect metrics from the platform components. Supported values are `full` or `minimal`. In the `full` profile (default) Prometheus will collect all the metrics that are exposed by the platform components. In the `minimal` profile Prometheus will only collect metrics necessary for the default platform alerts, recording rules, telemetry and console dashboards. |

Reviewer: ```suggestion
| collectionProfile | CollectionProfile | Defines the metrics collection profile that Prometheus uses to collect metrics from the platform components. Supported values are `full` or `minimal`. In the `full` profile (default), Prometheus collects all metrics that are exposed by the platform components. In the `minimal` profile Prometheus only collects metrics necessary for the default platform alerts, recording rules, telemetry and console dashboards. |
```
